_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -206,6 +206,95 @@ You will most likely not need to specify any parameters except for `location`. F
 
 You will most likely not need to specify any parameters except for `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/).
 
+
+### Azure storage account
+
+These steps explain how to register a snapshot repository backed by an Azure storage account for an OpenSearch cluster deployed using Helm.
+
+1. Create an Azure storage account. Then create a container within the storage account.
+
+1. Create a Docker image with the `repository-azure` plugin.
```

Comment:
```suggestion
1. Create an OpenSearch keystore for the Azure storage using the following command:
```

Reply:
@Naarcha-AWS the user will not be required to execute the bash script below to create the keystore. The user only needs to create the file. Then when the Docker image is built, the script will be copied into the image and executed. Therefore if we say `Create an OpenSearch keystore for the Azure storage using the following command:` it will give the impression that the user needs to create the keystore by executing the script. Instead shall we say `Create the following bash script`?